src/order/order.controller.spec.ts

```
@@ -0,0 +1,243 @@
+    const firstMember = new Member();
+    firstMember.id = v4();
+    firstMember.appId = app.id;
+    firstMember.email = 'owner@example.com';
+    firstMember.username = 'owner';
+    firstMember.role = role.name;
+    firstMember.name = 'firstMember';
+
+    const secondMember = new Member();
+    secondMember.id = v4();
+    secondMember.appId = app.id;
+    secondMember.email = 'receiver@example.com';
+    secondMember.username = 'receiver';
+    secondMember.role = role.name;
+    secondMember.name = 'secondMember';
+
+    const orderLog = new OrderLog();
+    orderLog.id = app.symbol + '111111111111111';
+    orderLog.member = firstMember;
+    orderLog.invoiceOptions = {};
+
+    it('should AuthToken is invalid', async () => {
+      const requestBody: TransferReceivedOrderBodyDTO = { token: '', memberId: undefined };
+      const requestHeader = {
+        authorization: 'Bearer ' + '',
+      };
+
+      await request(application.getHttpServer())
+        .put(apiPath.order.transferReceivedOrder)
+        .set(requestHeader)
+        .send(requestBody)
+        .expect(400)
+        .expect(/{"statusCode":400,"message":"E_TOKEN_INVALID"/);
+    });
+
+    it('should TransferOrderToken is invalid', async () => {
+      const tokenResponse = await request(application.getHttpServer())
+        .post(apiPath.auth.token)
+        .send({ clientId: 'test', key: 'testKey', permissions: [] });
+      const {
+        result: { authToken },
+      } = tokenResponse.body;
+
+      const requestBody: TransferReceivedOrderBodyDTO = { token: '', memberId: undefined };
+      const requestHeader = {
+        authorization: 'Bearer ' + authToken,
+      };
+
+      await request(application.getHttpServer())
+        .put(apiPath.order.transferReceivedOrder)
+        .set(requestHeader)
+        .send(requestBody)
+        .expect(400)
+        .expect(/{"statusCode":400,"message":"E_TOKEN_INVALID"/);
+    });
+
+    it('should orderId is invalid', async () => {
```

`Should raise error due to orderId is not found` ?